service/controller/legacy/v27/version_bundle.go

```diff
@@ -8,8 +8,8 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
+				Component:   "encrypter",
```

Comment:
What kind of encrypter is this actually? Would `aws-operator/encrypter` make more sense or something like `vault/encrypter`?